test/conditions.js

```diff
@@ -1094,60 +1094,34 @@ describe('Conditions', function(){
       type: 'select',
       table: 'users',
       where: {
-        $not: { email : "not-me@internet.com" }
+        email: { $not : "not-me@internet.com" }
       }
     })
 
     assert.equal(
       query.toString()
-      , 'select "users".* from "users" where not ("users"."email" = $1)'
+      , 'select "users".* from "users" where not "users"."email" = $1'
       );
     })
 
-  it('should handle $not correctly for expression with $or', function() {
+  it('should handle $not correctly for expression with other expression', function() {
     var query = builder.sql({
       type: 'select',
       table: 'users',
       where: {
-        $not: {
-        $or: [{
-          email: 'not-me@internet.com'
-        },
-        {
-          email: 'me-neither@internet.com'
-        }
+        age: {
+          $not: [{
+            $lte: 12
+          }
         ]
       }
     }
   })
 
   assert.equal(
     query.toString()
-    , 'select "users".* from "users" where not ("users"."email" = $1 or "users"."email" = $2)'
+    , 'select "users".* from "users" where not "users"."age" <= $1'
     );
   })
 
-  it('should handle $not correctly for expression with $and', function() {
```

Comment:
How come we remove this test?

Reply:
Because i don't believe it is correct to put the $not on the outside of logic operators like $and and $or.  $not should usually sit under the field name, similar to $gt / $lt, e.g.

```myField: {
    $not: someValue
}```

And and or logic can be used, but the not should sit under, e.g.

``` $and: [
            {
                  $not: someValue
             },
            {
                  $not: someOtherValue
             }
      ]```